### PR TITLE
Andrea 2

### PR DESCRIPTION
**Pull request by andreaalmeida**
**Removed npm and Github stars on mobile**
screen shot 2016-08-30 at 1 16 13 pm

**Added Github and Twitter to header bar, removed dropdown menu for Github**

screen shot 2016-08-30 at 1 16 31 pm

**Added npm and Github stars next to title on module pages**

screen shot 2016-08-30 at 2 06 17 pm

**NOTE:** 
The npm code and Github stars were added to the template page title.mustache.

In order for all module pages to work, you will need to add @package and @github tags to the documentation in docMap.json

screen shot 2016-08-30 at 2 02 54 pm

@justinbmeyer 
@tomgreever

### CODE REVIEW

File: static/title.less [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	line-height: 1.6;
+	margin-bottom: @gutter;
+    list-style-type: disc;
+    display: none;
+
+    @media screen and (min-width: @breakpoint) {
+    	display: unset;
+    }
+
+	> li{
```
Comment (tomgreever) [inline static/title.less]: Your spacing/indentation throughout here is really aggressive. :) You'll find that other devs will give you a hard time about stuff like this, so you might fix it all so it's more readable.

**Comment by tomgreever** [comment]
@andreaalmeida you'll need to check these on mobile and adjust accordingly (remembering that the code is mobile first and larger screens are the exception) - the main menu and the "social stuff" is off on smaller screens. 

screen shot 2016-08-30 at 2 11 19 pm
screen shot 2016-08-30 at 2 11 04 pm

**Comment by justinbmeyer** [comment]
@tomgreever 

> larger screens are the exception

This probably isn't true with a site like canjs.  Comparatively very few people visit on a mobile device.  

overview_-_analytics

**Comment by tomgreever** [comment]
That may be true of users and visitors, but the CSS is architected as mobile first. Regardless of how many people visit the site on mobile, we still want it to look good on mobile. My note was more a reminder for Andrea of how the CSS is written.

**Comment by andreaalmeida** [comment]
The Github and Twitter buttons show up in the header nav on desktop, and move to the sidebar menu on mobile.  

On mobile, the Github button needs to be replaced with Twitter - I kept getting an error from the script tag when I copied and pasted the Twitter button from the header nav.

**Comment by andreaalmeida** [comment]
screen shot 2016-09-02 at 3 45 31 pm
screen shot 2016-09-02 at 3 45 45 pm

**Comment by andreaalmeida** [comment]
@justinbmeyer here is the updated mobile view with a 2 x 2 grid for Github, Twitter, Chat, and Forum.

screen shot 2016-09-06 at 3 59 35 pm

**Pull request merged**